oauth2/oauth2-core/src/main/java/org/springframework/security/oauth2/core/ClaimAccessor.java

```diff
@@ -136,8 +136,9 @@ default URL getClaimAsURL(String claim) {
 	 * Returns the claim value as a {@code Map<String, Object>} or {@code null} if it does
 	 * not exist or cannot be assigned to a {@code Map}.
 	 * @param claim the name of the claim
-	 * @return the claim value or {@code null} if it does not exist or cannot be assigned
-	 * to a {@code Map}
+	 * @return the claim value or {@code null} if it does not exist
+	 * @throws IllegalArgumentException If the claim exists, but cannot be converted to a
```

Comment:
For consistency with the rest of the Javadoc, will you please leave "If" lowercased in both of your changes?